Alert/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="alert primary">
+      <strong> Primary!! </strong> This is a primary Alert
+    </div>
+
+    <div class="alert success">
+      <strong> Success!! </strong> This is a primary Alert
+    </div>
+
+    <div class="alert warning">
+      <strong> Warning!! </strong> This is a primary Alert
+    </div>
+
+    <div class="alert error">
+      <strong> Error!! </strong> This is a primary Alert
```

Comment:
All of your alerts say `this is a primary alert`. I would suggest changing the alert names to success, warning etc. as it will tell the user which is which

Reply:
Thanks Kartikeya for pointing this out i will fix this